hydroengine_service/dgds_functions.py

```diff
@@ -280,20 +310,25 @@ def generate_elevation_map(dataset_list=None):
     return info
 
 
-def visualize_gebco(source, band):
+def visualize_gebco(source, band, min=None, max=None):
     """
     Specialized function to visualize GEBCO data
     :param source: String, Google Earth Engine image id
     :param band: String, band of image to visualize
     :return: Dictionary
     """
     data_params = DATASETS_VIS[source]
+    if min:
+        data_params['bathy_vis_params']['min'] = min
+    if max:
```

Comment:
if max is not None
See comment on line 584 - # min/max values can be zero, should not be None